buildSrc/src/main/java/Versions.kt

```diff
@@ -22,7 +22,7 @@ object Versions {
     const val lifeCycle = "2.5.1"
     const val compose_compiler_ext = "1.5.3"
     const val compose_material3 = "1.1.2"
-    const val compose_version = "1.5.3"
+    const val compose_version = "1.7.2"
```

Comment:
Shall we use the latest stable release [1.7.4](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.4)?